README.md

```diff
@@ -11,26 +11,30 @@
 
-With the advent of Virtual and Augmented reality, a new style of 3D interfaces is emerging. We are no longer limited to flat screens; instead the user interface can be broken up in sections and positioned within your 3D space (for VR) or the real world (for AR).
+With the advent of virtual and augmented reality, a new style of 3D interfaces is emerging.
+We are no longer limited to flat screens; instead the user interface can be broken up in sections and positioned within the user's 3D space (for VR) or the real world (for AR).
 
 Browser vendors are developing WebXR for WebGL rendered content and are experimenting with [basic 3D models](https://developers.google.com/web/updates/2019/02/model-viewer) and other [interactive experiences](https://creator.magicleap.com/learn/guides/prismatic-getting-started).
-However, until now it was not possible to position CSS generated content in 3D.
+However, until now, positioning CSS generated content in 3D was not possible.
 
-## proposal
+## Proposal
 
-Our goal is to break off parts of web page into the 3-d space around the browser surface. 
+Our goal is to break off parts of web page into the 3D space around the browser surface. 
 
-This would make viewing web pages a much more immersive experience rather than looking at a rectangular surface. We hope to do accomplish this by introducing a small addition to CSS.
+This would make viewing web pages a much more immersive experience rather than looking at a rectangular surface.
+We hope to accomplish this by introducing a small addition to CSS.
 
 We propose to introduce a new value for `transform-style`: `detached`.
 
-This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`. The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
+This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`.
+The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
 If `transform-style: detached` is specified on an element, children with 3D transforms will be lifted out of the page.
 
 Here is an example rendering of the effect:
 ![scene](https://github.com/rcabanier/detached_explainer/raw/master/detached.gif "Scene")
 
-As the page or a parent element scrolls, the detached element should scroll. CSS animations and transitions will apply as well.
+As the page or a parent element scrolls, the detached element should scroll.
+CSS animations and transitions will apply as well.
 
 Because we don't want to render content anywhere in the user's space, we also propose to define a "safe space" around the browser.
```

Comment:
Prefer: In order to disallow unrestricted access of user's space to the content developer, we also propose to define a "safe space" that shall be requested to the user by content developer and user consented.

Reply:
updated.